server/engine/src/main/java/org/eclipse/lsp/cobol/lsp/SourceUnitGraph.java

```diff
@@ -132,6 +135,35 @@ private synchronized void updateDocumentGraphUponCompletion(
     documentGraph.put(model.getUri(), references);
   }
 
+  private void removePreviousMapping(String uri) {
```

Comment:
I'm a bit confused with the name of the method. What mapping is it?

Reply:
renamed to invalidateNodeLinks. This method removes prev links with other nodes.